### PR TITLE
[#8969][Platform] [API] [UI] API portal whitelabeling 

### PR DESCRIPTION
**Pull request by jaydeepkumara**
Summary:- This is for whitelabeling and any other look and feel changes for swagger-ui available on /docs/
The assets are currently checked in under /managed/ui/public/swagger-ui/
New UI look and feel:- https://stoplight.io/open-source/elements/
Implementation :- https://meta.stoplight.io/docs/elements/docs/getting-started/usage/web-component.md

Test Plan:- 
1. Open Swagger UI. try to create, update, list etc.
2. Validate operationId for all API are differ as I use operationId to disguise API.

**New Swagger UI screenshot**

Screenshot 2021-07-09 at 10 06 26 PM

**Delete User API**

Screenshot 2021-07-10 at 1 22 17 AM

**Add User API**
Screenshot 2021-07-10 at 1 21 34 AM

### CODE REVIEW

File: managed/src/main/java/com/yugabyte/yw/controllers/CloudProviderController.java [hunk 1]
```diff
@@ -42,6 +42,7 @@
    *
    * @return JSON response of newly created provider
    */
+  @ApiOperation(value = "createCloudProvider", nickname = "createCloudProvider")
```
Comment (sb-yb) [inline managed/src/main/java/com/yugabyte/yw/controllers/CloudProviderController.java:45]: I think someday I can change this to generate operationId automatically. For now lets live with this.

**Comment by sb-yb** [comment]
This looks much better that swagger-UI. 
High level question: 
  Is there a way to add this as a library dependency and configure it?
For example with swagger-UI I was going to start using the library and only add swagger-config.yaml. See https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/configuration.md

Do we have equivalent so we don't have to add ton of third party code to our repo.

File: managed/ui/public/swagger-ui/index.html [hunk 2]
```diff
@@ -4,74 +4,23 @@
...
-        deepLinking: true,
-        presets: [
-          SwaggerUIBundle.presets.apis,
-          SwaggerUIStandalonePreset
-        ],
-        plugins: [
-          SwaggerUIBundle.plugins.DownloadUrl
-        ],
-        layout: "StandaloneLayout",
-        requestInterceptor: function(req) {
```
Comment (sb-yb) [inline managed/ui/public/swagger-ui/index.html:66]: See this. I had fixed the CSRF issue so swagger-ui could be used to try out post and put requests as well. 
Does stoplight take care of it internally?

**Review by sb-yb — CHANGES_REQUESTED** [review]
Just one comment about applying csrf fix.

File: managed/ui/public/swagger-ui/index.html (hunk 2 above)
Comment (jaydeepkumara) [reply managed/ui/public/swagger-ui/index.html:66]: Yes. I have tested delete and post API.

Comment (jaydeepkumara) [reply managed/ui/public/swagger-ui/index.html:66]: Worked Added screenshot.

**Review by sb-yb — APPROVED** [review]

**Pull request merged**